src/pages/static/js/tabs.js

```diff
@@ -36,16 +36,18 @@ displayTabs(tabsData) {
     });
   }
 
-  groups.set('ungrouped', { id: 'ungrouped', name: 'Ungrouped', color: '#6b7280', expanded: true, tabs: [] });
-
   tabsData.tabs.forEach(tab => {
-    const gId = tab.groupId && groups.has(tab.groupId) ? tab.groupId : 'ungrouped';
-    if (!groups.has(gId)) {
-      groups.set(gId, { id: gId, name: 'Group', color: '#6b7280', expanded: true, tabs: [] });
+    if (tab.groupId && groups.has(tab.groupId)) {
+      groups.get(tab.groupId).tabs.push(tab);
     }
-    groups.get(gId).tabs.push(tab);
   });
 
+  const hasGroupsWithTabs = Array.from(groups.values()).some(group => group.tabs.length > 0);
+  if (!hasGroupsWithTabs) {
+    container.innerHTML = '<p>No tab groups found.</p>';
+    return;
+  }
```

Comment:
It's showing no tab groups found even after creating a group:
<img width="912" height="712" alt="Screenshot 2025-08-02 at 4 08 04 PM" src="https://github.com/user-attachments/assets/fdc7709c-7be7-417a-98c9-2283bc667b7f" />